source/CommonJobs/CommonJobs.MVC.UI/Scripts/Evaluations/calification.js

```diff
@@ -285,6 +286,14 @@
                             text: itemNumber + " - " + item.Text,
                             description: item.Description,
                             isRowSelected: rowSelected,
+                            addSelfComment: function (data, event) {
+                                _.find(this.comments, function (selfComment) {
```

Comment:
I see. Thanks! 
Then, the predicate should be `selfComment.evaluatorEmployee == self.userLogged` and then work with the comment I get